code/modules/surgery/organs/organ_internal.dm

```diff
@@ -98,13 +98,16 @@
 
 //Checks to see if the organ is frozen from temperature
 /obj/item/organ/proc/is_cold()
-	var/freezing_objects = list(/obj/structure/closet/crate/freezer, /obj/structure/closet/secure_closet/freezer, /obj/structure/bodycontainer, /obj/item/autosurgeon)
+	var/freezing_objects = list(/obj/structure/closet/crate/freezer, /obj/structure/closet/secure_closet/freezer, /obj/structure/bodycontainer, /obj/item/autosurgeon, /obj/machinery/smartfridge/organ)
 	if(istype(loc, /obj/))//Freezer of some kind, I hope.
 		if(is_type_in_list(loc, freezing_objects))
```

Comment:
`is_type_in_typecache(loc, freezing_objects)` will suffice now that we are using a typecache.